### ISSUE TITLE
Cannot exit Dexed

### ISSUE THREAD

**Issue by vporton** [BBasile/dexed#392]
When I try to exit from Dexed (by clicking the window close icon), the window does not close and I get this message:

```
Unable to create file "/home/porton/.config/dexed/docking.xml.tmp"
```

Dexed 3.7.0

Ubuntu 18.10

**Comment by BBasile** [comment]
There must be something wrong with the authorizations on your system. I'm afraid i can't help it.

**Comment by vporton** [comment]
Yes, `~/.config/dexed` was owner by root.

**Comment by BBasile** [comment]
I remember that it already happened to someone. He installed dexed using sudo , then in the same console launched dexed, with elevated privileges. Then necessarily settings were stored for root.

**Comment by FR86** [comment]
Ran into this today. I've been downloading the 64 bit linux setup from [here](https://github.com/Basile-z/dexed/releases/download/v3.7.5/dexed.3.7.5.linux64.setup.zip). I then followed [these](https://basile-z.github.io/dexed/setup.html) instructions and ran the unzipped setup sudo'ed.
Afterwards, I started dexed from the same terminal session, but for sure without sudo.
The ```~\.config\dexed``` folder was created with root ownership. Changing that with ```sudo chown myuser:myuser -R /home/myuser/.config/``` fixed the issue, but I wonder how it got created like that in the first place...

**Comment by Basile-z** [comment]
can it be because of that ?

https://github.com/Basile-z/dexed/blob/41881be4991df726bdfbe22e8db674de645ee6e4/setup/setup.d#L324-L325

This execute for each binary of the installer since unlike for an archive the execution flag is not maintained.

**Comment by vporton** [comment]
No, this is apparently because of the file UID/GID (`root`) not because file permissions set by `chmod`.

UID/GID is set by the Linux command `chown` not by `chmod`.

Your `chmod` command seems correct for executable files, but for all files (both executable and not) there should _also_ be run a suitable `chown`.

However, I don't know where you can obtain the correct UID and GID.

Notwithstanding the above, I recommend the setup program when started with `sudo` not to write anything to the user directory at all! It should be done by Dexed itself (not its installer), because Dexed runs as a user process not a root process and so cannot damage UID and GID as your installed seems to do.

I don't remember, maybe you run Dexed from the installer? Then it should be run under correct UID/GID.

Easy solution: Don't start Dexed from the installer. It is an acceptable and yet easy solution.

**Comment by Basile-z** [comment]
Finally fixed in fea5f7e4ce98bf38936f1ce35bebd90e191b08ff.
